service.subsmangler/resources/lib/pysubs2/__main__.py

```diff
@@ -0,0 +1,7 @@
+import sys
```

Comment:
Please do not bundle python modules inside your addon. Can you submit pysubs2 as a script.module? Is it plain upstream or have you patched the module?

Reply:
At the moment it is a plain upstream. This module is borrowed from https://github.com/tkarabela/pysubs2
At the moment there is no Kodi pysubs2 module I know of. As I'm not the author of this module, I wanted to avoid maintaining it as there can be other plugins can be based on it in the future.

Reply:
you don't need to maintain it (if it is plain upstream) but we do request to have it as a separate module